_config/config.yml

```diff
@@ -19,7 +19,7 @@ Seeder:
     - URLSegmentProvider
     - ValueProvider
     # tests
-    - LittleGiant\SilverStripeSeeder\Tests\TestProvider
+    - TestProvider
```

Comment:
I think that we should prefix these with 'Seeder' if they aren't namespaced, the potential for conflicts is pretty massive. Especially 'SortProvider' etc